go/registry/api/grpc.go

```diff
@@ -26,7 +26,9 @@ var (
 	// methodGetNodeStatus is the GetNodeStatus method.
 	methodGetNodeStatus = serviceName.NewMethod("GetNodeStatus", IDQuery{})
 	// methodGetNodes is the GetNodes method.
-	methodGetNodes = serviceName.NewMethod("GetNodes", int64(0))
+	methodGetNodes = serviceName.NewMethod("GetNodes", IDsQuery{})
```

Comment:
Uh, this can cause some weird breakage.

Reply:
Wasn't sure which components and how many use this endpoint. I thought that renaming was a better solution than using name `NodesByID`.

Reply:
The grpc API changes should probably have a breaking changelog fragment since external services use it to communicate with the node (e.g. oasis-cli, oasis-wallet, rosetta-gateway...), there also is a public deployment of it.

Reply:
I would try to avoid breakage if possible.

Reply:
Removed commit that renamed grpc methods and changed `fetchKeys` (in commit 
_go/keymanager: Move helper functions to a separate package_) to query nodes one by one like we had before.